services/kuber/templates/autoscaler-adapter.goyaml

```diff
@@ -3,9 +3,13 @@ apiVersion: v1
 kind: Service
 metadata:
   name: autoscaler-adapter-{{ .ClusterID }}
+  labels:
+    app.kubernetes.io/part-of: claudie
+    app.kubernetes.io/name: autoscaler-adapter-{{ .ClusterID }}
```

Comment:
I think this shouldn't be templated, it should just be `autoscaler-adapter`
But actually, instead of that,  I'd go with `app.kubernetes.io/name: cluster-autoscaler` for everything